lib/vsc/install/ci.py

```diff
@@ -201,19 +201,30 @@ def indent(line, level=1):
         'python2.7 -V',
     ]
 
+    if vsc_ci_cfg[INSTALL_SCRIPTS_PREFIX_OVERRIDE]:
+        pip_args = '--install-option="--install-scripts={envdir}/bin" '
+        easy_install_args = '--script-dir={envdir}/bin '
+    else:
```

Comment:
I'd prefer this be the default. It's hard to see that this will exist in the below if.

Reply:
~~~~
pip_args = ''
easy_install_args = ''
if vsc_ci_cfg[INSTALL_SCRIPTS_PREFIX_OVERRIDE]:
    pip_args += '--install-option="--install-scripts={envdir}/bin" '
    easy_install_args += '--script-dir={envdir}/bin '
~~~~